src/ctia/task/settings.clj

```diff
@@ -0,0 +1,57 @@
+(ns ctia.task.settings
+  (:require [clojure.tools.cli :refer [parse-opts]]
+            [clojure.tools.logging :as log]
+
+            [schema.core :as s]
+            [clj-momo.lib.es
+             [index :as es-index]
+             [schemas :refer [ESConnState]]]
+            [ctia.stores.es.init :refer [upsert-template!]]
+            [ctia
+             [init :refer [init-store-service! log-properties]]
+             [properties :refer [properties init!]]
+             [store :refer [stores]]]))
+
+(s/defn update-store!
+  "read store properties of given stores and update indices settings."
+  [{:keys [conn index config]
+    {:keys [settings]} :config
+    :as state} :- ESConnState]
+  (try
+    (upsert-template! conn index config)
+    (log/info "updated template: " index)
+    (->> {:index (select-keys settings [:refresh_interval :number_of_replicas])}
+         (es-index/update-settings! conn index))
+    (log/info "updated settings: " index)
+    (catch clojure.lang.ExceptionInfo e
```

Comment:
Could you import `clojure.lang.ExceptionInfo` instead of using the full classname?